### PR TITLE
update outdated yaml

### PR DESCRIPTION
**Pull request by tanjunchen**
kubectl version
```
Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.0", GitCommit:"70132b0f130acc0bed193d9ba59dd186f0e634cf", GitTreeState:"clean", BuildDate:"2019-12-07T21:20:10Z", GoVersion:"go1.13.4", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.0", GitCommit:"70132b0f130acc0bed193d9ba59dd186f0e634cf", GitTreeState:"clean", BuildDate:"2019-12-07T21:12:17Z", GoVersion:"go1.13.4", Compiler:"gc", Platform:"linux/amd64"}
```
/cc  @stefanprodan 
I have tested it in v1.17.0,but i found that some yamls are wrong,so they are needed to improved.
the repository is popular,so i think we can update there,Thanks
could you think about it？

### CODE REVIEW

File: podinfo/podinfo-hpa-custom.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
-apiVersion: autoscaling/v2beta2
+apiVersion: autoscaling/v2beta1
```
Comment (stefanprodan) [inline podinfo/podinfo-hpa-custom.yaml]: Why the downgrade?

File: Makefile [hunk 2]
```diff
@@ -42,10 +42,3 @@ endif
 rmcerts:
 	@rm -f apiserver-key.pem apiserver.csr apiserver.pem
 	@rm -f metrics-ca-config.json metrics-ca.crt metrics-ca.key
-
-.PHONY: deploy
```
Comment (stefanprodan) [inline Makefile]: Please undo the makefile changes, they are not related to this PR.

Comment (tanjunchen) [reply Makefile]: ok

File: podinfo/podinfo-hpa-custom.yaml (hunk 1 above)
Comment (tanjunchen) [reply podinfo/podinfo-hpa-custom.yaml]: see https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#autoscaling-on-multiple-metrics-and-custom-metrics

Comment (stefanprodan) [reply podinfo/podinfo-hpa-custom.yaml]: The docs are with v2beta2 and you've changed to v2beta1, why?

Comment (tanjunchen) [reply podinfo/podinfo-hpa-custom.yaml]: Very few changes Haha

**Pull request closed without merging**